### ISSUE TITLE
There is no page definition in this test context

### ISSUE THREAD

**Issue by mselmansezgin** [BuseSolmaz/RabbitMQTest#1]
Page package and BasePage class namings are irrelevant. Please use proper naming for both package and class.

**Comment by BuseSolmaz** [comment]
Yes, I noticed after I committed the project. Now it is done, you can check it.
